README.md

```diff
@@ -59,9 +58,34 @@ the [kerl](https://github.com/kerl/kerl) build system, and installs them to
 `/usr/local/kerl` for activation before builds. This version is intended for use
 in standard CI runs, such as for pull requests.
 
-# Building a cross-architecture Docker image
+## Building images for other architectures
+
+### Multi-arch images with Docker Buildx
+
+We can use Docker's
+[Buildx](https://docs.docker.com/buildx/working-with-buildx/) plugin to generate
```

Comment:
A tiny nit, but since it tripped me up, it might be worth mentioning that the open source docker.io package from Ubuntu default repositories do not support buildx (at least in LTS 20.04). So to use it, it is necessary to directly install the package from docker.io. Overall I am not particularly keen on standardizing on proprietary bits from Docker.

reference: https://docs.docker.com/buildx/working-with-buildx/#linux-packages

Reply:
It turns out the non-buildx way of building the multi-arch images doesn't work any longer. I gave up and installed the docker.io docker with buildx. 

So +1 to merge this

Reply:
It turns out the non-buildx way of building the multi-arch images doesn't work any longer. I gave up and installed the docker.io docker with buildx. 

So +1 to merge this